.github/workflows/release.yaml

```diff
@@ -47,21 +49,27 @@ jobs:
             exit 1
           fi
 
-      - name: Extract current release version through tag
+      - name: Extract current release version through tag and set SEMVER_VERSION
         id: get_tag_version
         run: |
           tag=$(git describe --tags --abbrev=0)
           semver=${tag#v}  # Remove the 'v' prefix from version number
           echo "Current Released Version: $semver"
-          echo "semver=$semver" >> $GITHUB_ENV # Set as environment variable
+          echo "SEMVER_VERSION=$semver" >> $GITHUB_ENV # Set as environment variable
+
+      - name: Set PRERELEASE_SUFFIX if isPrerelease is true
+        id: set_prerelease_suffix
+        if: ${{ inputs.isPrerelease == true }}
+        run: |
+          echo "PRERELEASE_SUFFIX=${{ inputs.prereleaseSuffix }}" >> $GITHUB_ENV
 
       - name: Bump version
         id: bump
         uses: "./.github/actions/bump-version"
         with:
-          versionFile: pinecone/__version__
+          currentVersion: ${{ env.SEMVER_VERSION}}
```

Comment:
Copy/pasta mistake from the original workflow I used as a template. The underlying `bump-version` action seems to work as expected and is unit tested, I just wasn't passing the proper input.